src/bokeh/models/tools.py

```diff
@@ -531,6 +534,20 @@ def __init__(self, *args, **kwargs) -> None:
     wheel pan tool will pan the plot along the x-axis.
     """)
 
+    modifiers = Modifiers(default={}, help="""
+    Allows to configure a combination of modifier keys, which need to
+    be pressed during the selected gesture for this tool to trigger.
```

Comment:
I think an example code block for setting this property would be extremely helpful (here and elsewhere)